[仮想環境でpipをアップグレードしたらぶっ壊れった。]
内容：
> pip install --upgrade pip
Collecting pip
  Downloading https://files.pythonhosted.org/packages/8d/07/f7d7ced2f97ca3098c16565efbe6b15fafcba53e8d9bdb431e09140514b0/pip-19.2.2-py2.py3-none-any.whl (1.4MB)
     |████████████████████████████████| 1.4MB 656kB/s
Installing collected packages: pip
  Found existing installation: pip 19.2.1
    Uninstalling pip-19.2.1:
ERROR: Could not install packages due to an EnvironmentError: [WinError 5] アクセスが拒否されました。: 'd:\\python\\python36\\scripts\\pip.exe'
Consider using the `--user` option or check the permissions.

解決方法：
    仮想環境の作り直し
別解：
    https://qiita.com/yasthon/items/954dc1aef9752838a0f0